src/app/[locale]/(pages)/blocco-accesso/page.tsx

```diff
@@ -38,8 +38,8 @@ const ProfileBlock = (): React.ReactElement => {
       });
   };
 
-  const renderSummary = (isIDPKnown: boolean) => {
-    if (isIDPKnown) {
+  const renderSummary = (isIDPKnown?: string) => {
+    if (isIDPKnown === 'true') {
```

Comment:
What about making an utility function to parse string 'true'/'false' present in our .env* files, and use it in the exported flags?